internal/app/orchestrator/orchestrator.go

```diff
@@ -98,12 +99,18 @@ func New(
 		orchestrator.logger.With("error", err).Panic(ctx, "failed to initialize cache")
 	}
 	orchestrator.cache = cache
+	http.HandleFunc("/cache/", orchestrator.CacheDumpHandler)
```

Comment:
What are your thoughts on moving the handlers to it's own package? `internal/app/http/handler` or whatnot? I haven't looked into the best practices for this, but I'd prefer to keep the orchestrator contained to the request/response management flow.

Reply:
I was concerned about the accessbility of the resources (the cache specifically) but I'll see if I can move everything out into a separate package.

Reply:
I moved things around in my latest commit, including the cache dump handler (and a first attempt at just doing the Fetch based on a key passed in). I had to create an accessor for the orchestrator cache. LMK what you think of this approach versus registering the handler in the orchestrator.

Reply:
One advantage of having this handler live with the orchestrator is that the orchestrator has access to all the cache keys (I think this is the case based on the discussion around DeleteWatch) so it makes implementing a full cache dump easier.

Reply:
I'm in favor of the new approach that you have. Something else worth considering is rather than exposing the entire cache via the orchestrator interface, to just add a read only function so that we're not accidentally exposing writes elsewhere. i.e., return a copy of the fetched results, including a copy of the keys so we don't return the original pointers. Thoughts?

Reply:
I agree with having it be read-only and added a "read-only cache" interface that only exposes Fetch.